config/serverless.yml

```diff
@@ -274,6 +274,7 @@ xpack.reporting.csv.maxConcurrentShardRequests: 0
 # Disabled Platform plugins
 monitoring.enabled: false
 monitoring.ui.enabled: false
+monitoring_collection.enabled: false
```

Comment:
__question:__ If it's something that _fundamentally_ shouldn't be enabled in Serverless, shouldn't we set it to `false` directly in the code using `offeringBasedSchema` to not spread that knowledge across multiple sources?

Reply:
>I thought that you referred to not allowing it to be enabled on Serverless. 

We could do that too (ideal) if we were 100% sure we'd never need to enable it to not leave ambiguous interpretation while reading code in the future ("oh, I see there is no way this functionality can be enabled in Serverless, ignoring" vs "hmm, it looks like it can theoretically be enabled in Serverless, but why? Let me read the code, check gitops overrides, ask Core Team" 🙂